main.go

```diff
@@ -124,7 +125,7 @@ func main() {
 	if err = (&group.GroupReconciler{
 		Client:            mgr.GetClient(),
 		Scheme:            mgr.GetScheme(),
-		MetricsAggregator: metrics.GetMetricsAggregator(),
+		MetricsAggregator: metrics.GetMetricsAggregator(clusterId),
```

Comment:
Make sure all the reconcilers you added the `ClusterId` field to get populated, e.g.

```suggestion
		MetricsAggregator: metrics.GetMetricsAggregator(clusterId),
		ClusterId:                  clusterId,
```